frontend/components/card/search-result/CardSearchResultUser.vue

```diff
@@ -54,6 +54,7 @@ import type { User } from "~/types/user";
 
 defineProps<{
   isPrivate?: boolean;
+  reduced?: boolean;
```

Comment:
Looks like the changes weren't applied to user search results? I can get these :)